recipes/configure.rb

```diff
@@ -14,7 +14,7 @@
     mode "0660"
     group deploy[:group]
     owner deploy[:user]
-    variables(:database => deploy[:database], :environment => deploy[:rails_env])
+    variables(:database => deploy[:database], :environment => deploy[:rails_env], :deploy => deploy)
```

Comment:
@johncblandii Not sure how you feel about it, but it seems like that commit may not be relevant to this PR?